pkg/plugin/twinmaker/client_test.go

```diff
@@ -186,7 +186,7 @@ func TestFetchAWSData(t *testing.T) {
 	})
 
 	t.Run("athena connector test no filter", func(t *testing.T) {
-		// t.Skip()
+		t.Skip()
```

Comment:
this was failing locally. i assume it should be skipped since it requires credentials?

Reply:
yes this should be skipped